pkg/cluster/internal/providers/docker/network.go

```diff
@@ -271,7 +271,10 @@ func isIPv6UnavailableError(err error) bool {
 	// even on hosts that lack ip6tables setup.
 	// Preferably users would either have ip6tables setup properly or else disable ipv6 in docker
 	const dockerIPV6TablesError = "Error response from daemon: Failed to Setup IP tables: Unable to enable NAT rule:  (iptables failed: ip6tables"
```

Comment:
We should maybe rename and clarify this one versus the new error. That shouldn't block fixing the bug though